src/main/java/com/ttbkk/api/domain/hashtag/Hashtag.java

```diff
@@ -0,0 +1,30 @@
+package com.ttbkk.api.domain.hashtag;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+//Entity 클래스
+@Getter
+@NoArgsConstructor
+@Entity
+@Table(name = "hashtag")
+public class Hashtag {
+
+    @Id
+    @Column(columnDefinition = "VARCHAR(150)")
+    private String name;
+
+    @NotNull
+    @Column(name = "created_at", columnDefinition = "DATETIME(6)")
+    private LocalDateTime createdAt;
+
+    public Hashtag(String name) {
+        this.name = name;
+        this.createdAt = LocalDateTime.now();
+    }
```

Comment:
createdAt은 db에서 추가해줄거라 여기에선 넣지 않아도 될듯합니다
```suggestion
    public Hashtag(String name) {
        this.name = name;
    }
```

Reply:
https://github.com/ttbkk-univ/ttbkk-spring/pull/7/commits/6ae91281d929fe9ba75119f7bb9230a74366efc0